### PR TITLE
homework 6

### PR DESCRIPTION
**Pull request by Tobu9009**

### CODE REVIEW

File: Pages/Hotels/homework6Functions.js [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+        return month;
+    }
+    else if(monthNumber == 10){
+        const month = 'October'
+        return month;
+    }
+    else if(monthNumber == 11){
+        const month = 'November'
+        return month;
+    }
```
Comment (ahsansiddiki) [inline Pages/Hotels/homework6Functions.js:71]: I believe you can get above job done using moment library.

File: Pages/Hotels/homework6Functions.js [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+        return destinationOnSearch.toLowerCase().includes(userDestination.toLowerCase());
+    }
+    async getCheckInDate(){
+        return await this.commands.getElementText(this.checkInDate)
+    }
+    async getCheckOutDate(){
+        return await this.commands.getElementText(this.checkOutDate)
+    }
+    async getTomorrowDate(){
+        let tomorrow = now.subtract(5, 'days').format('M/D/Y')
```
Comment (ahsansiddiki) [inline Pages/Hotels/homework6Functions.js:164]: Not sure why you subtracting 5 days to get tomorrow.

File: features/Hotels/homework6.feature [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        Given I am on hotels landing page
+        When I click on date form
+        Then I verify that past dates of current month is not enabled
+    @homework6-2
+    Scenario: Verify destination and check-in and check-out dates are as user selected
+        Given I am on hotels landing page
+        When I click going to 
+        And I enter "man" in going to
+        And I select "Manila" from auto-suggestion
+        When I click on date form
```
Comment (ahsansiddiki) [inline features/Hotels/homework6.feature:13]: When => And
also in L16

**Review by ahsansiddiki — APPROVED** [review]
Left some comments for future improvements.

**Comment by Tobu9009** [comment]
I had to do this because I was confused on why the moment library begins
the next add/subtract date fetch based on the last add/subtract date fetch.
For example it is currently today, I then want to get 5 days from today so
I add 5 days, moment will get me to 9/9/22, however, lets say I want to get
yesterdays date next, if I do subtract 1, it gets me 9/8/22 not 9/3/22. So
I have to subtract 6 days to get yesterdays date after I added 5 days
first.

**Pull request merged**